__tests__/utils.test.ts

```diff
@@ -11,17 +11,29 @@ import {
 } from '../src/utils';
 import { HIDDEN_MARKER } from '../src/constants';
 
-describe('isBotPr()', () => {
-  it('should return true for dependabot PR', () => {
-    expect(isBotPr('dependabot')).toBeTruthy();
+jest.spyOn(console, 'log').mockImplementation();
```

Comment:
why are we doing this here?

Reply:
oh got it :+1:

Reply:
do we need to do this if we aren't checking for console.log execution

Reply:
Just didn't like seeing console.logs in the output of tests :P